videointelligence/video_analyze/video_analyze.go

```diff
@@ -16,6 +16,7 @@ import (
 	"golang.org/x/net/context"
 )
 
+// [START video_analyze_labels_local]
```

Comment:
Would you mind adding a newline between the region tag comments and the function declaration? As is, the region tag comments will end up in the function GoDoc.

Reply:
How does that look?

Reply:
LGTM. I ran `gofmt` since it was complaining.